shared/common-adapters/relative-popup-hoc.desktop.js

```diff
@@ -245,11 +245,21 @@ function ModalPositionRelative<PP>(
     }
 
     render() {
+      // React will complain if WrappedComponent is a HTMLElement and we try to attach these props
```

Comment:
In `FloatingBox`, `WrappedComponent` is a div, so react complains about these unknown attributes.